### PR TITLE
Add new animation

### PR DESCRIPTION
**Pull request by ghost**
When player openInventory or closeInventory it add new animation. 

You can change it from 110-122 trigger to -> 445/458 the animation from here https://alexguirre.github.io/animations-list/.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -438,3 +440,19 @@ AddEventHandler(
...
+    RequestAnimDict( dict )
+    Citizen.Wait( 5 )
+    end
+    end
+
+    RegisterNetEvent('randPickupAnim')
+    AddEventHandler('randPickupAnim', function()
+    loadAnimDict('pickup_object')
+    TaskPlayAnim(PlayerPedId(),'pickup_object', 'putdown_low',5.0, 1.5, 1.0, 48, 0.0, 0, 0, 0)
+    Wait(1000)
```
Comment (iSentrie) [inline client/main.lua:456]: This makes loading so long.

**Review by iSentrie — CHANGES_REQUESTED** [review]
Improve loading time.

**Comment by ghost** [comment]
Hmmm, why ? > Improve loading time.

**Comment by iSentrie** [comment]
> Hmmm, why ? > Improve loading time.

Because it's taking too long and it's awkward decrease in performance, feels like simple lag.

File: client/main.lua (hunk 1 above)
Comment (ghost) [reply client/main.lua:456]: Then ->  

 AddEventHandler('randPickupAnim', function()
    loadAnimDict('pickup_object')
    TaskPlayAnim(PlayerPedId(),'pickup_object', 'putdown_low',3.0, 1.5, 1.0, 30, 0.0, 0, 0, 0)
    Wait(500)

**Pull request closed without merging**